end-to-end-tests/src/main/java/com/amazonaws/blox/integ/DynamoDBSupport.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may
+ * not use this file except in compliance with the License. A copy of the
+ * License is located at
+ *
+ *     http://aws.amazon.com/apache2.0/
+ *
+ * or in the "LICENSE" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+package com.amazonaws.blox.integ;
+
+import java.util.Map;
+import java.util.Map.Entry;
+import java.util.Set;
+import java.util.stream.Collectors;
+import lombok.RequiredArgsConstructor;
+import software.amazon.awssdk.services.dynamodb.DynamoDBClient;
+import software.amazon.awssdk.services.dynamodb.model.AttributeValue;
+import software.amazon.awssdk.services.dynamodb.model.DeleteItemRequest;
+import software.amazon.awssdk.services.dynamodb.model.DescribeTableRequest;
+import software.amazon.awssdk.services.dynamodb.model.DescribeTableResponse;
+import software.amazon.awssdk.services.dynamodb.model.KeySchemaElement;
+import software.amazon.awssdk.services.dynamodb.model.ScanRequest;
+import software.amazon.awssdk.services.dynamodb.model.ScanResponse;
+
+@RequiredArgsConstructor
+public class DynamoDBSupport {
+
+  private final DynamoDBClient ddb;
+
+  void truncateTable(String tableName) {
+    // This will only delete one page of records. Fine for now, but we should probably just use
```

Comment:
why are you not using the mapper now?

Reply:
Dependency problems. In order to access the data service repository models, I'd have to add a test dependency on the `data-service` top-level project. This pulls in a bunch of dependencies that weren't playing nice when combined with the other dependencies here, because of the spring-dependency-management plugin. I think we can revisit it once we've done the build logic cleanup to consolidate all of those; will update the comment to reflect that.